src/main/java/org/openmf/mifos/dataimport/handler/AbstractDataImportHandler.java

```
@@ -51,6 +61,16 @@ protected String readAsLong(int colIndex, Row row) {
         	Cell c = row.getCell(colIndex);
         	if (c == null || c.getCellType() == Cell.CELL_TYPE_BLANK)
         		return "";
+        	FormulaEvaluator eval = row.getSheet().getWorkbook().getCreationHelper().createFormulaEvaluator();
+        	if(c.getCellType() == Cell.CELL_TYPE_FORMULA) {
+        		CellValue val = null;
+        		try {
+        			val = eval.evaluate(c);
```

Even though Cell c is not null, bug in POI throws NPE if formula cell evaluates to null instead of returning null.